feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -64,7 +66,11 @@ export class AddToCartComponent implements OnInit, OnDestroy {
   @ViewChild('addToCartDialogTriggerEl') addToCartDialogTriggerEl: ElementRef;
 
   maxQuantity: number;
+
+  sapUnit: string;
+
```

Comment:
[MEDIUM]
Do we _really_ need to assign the `sapUnit` to a static property of the class?

If not, I suggest to remove the static property `sapUnit`, and just read the property `product?.sapCode?.sapUnit` directly from the `product` argument passed to the method `setStockInfo(product: Product)`

Reply:
removed